_pkgdown.yml

```diff
@@ -19,6 +19,7 @@ reference:
   contents:
     - select_records
     - get_record_option
+    - set_record_option
```

Comment:
This section has the title "Selection helpers", which doesn't fit `set_record_option`.  The title should either be broadened or this should be moved to its own section.

It'd also be nice to mention this in the getting started vignette, although I can do that later in a separate PR if you'd like.

Reply:
> It'd also be nice to mention this in the getting started vignette

I agree, and I'd vote to just put it in here. I'm always in favor of updating all the relevant docs with the code changes.

Reply:
@seth127 Just to confirm based on our earlier discussion (specifically the `"list1"` nuance), I should also add something about modifying the column names right?

I have a small section on it right now. 
<details closed>
<summary>New section on tables</summary>

<br>

Note that some options are position dependent (such as a data path 
or column names in `$TABLE` records), and have specific names for 
referencing their option (`"filename"` and `"list1"` respectively).
Column names are unique from other options, in that they are all 
concatenated and must be modified together:

```{r}
tbls <- nmrec::select_records(ctl, "table")
tbl1 <- tbls[[1]]
tbl1
tbl1_cols <- nmrec::get_record_option(tbl1, "list1")
tbl1_cols
tbl1_cols$value <- "ID TIME PRED RES WRES CPRED CWRES EPRED ERES EWRES PRED2"
tbl1_cols
```
</details>


@kyleam the main new section could be something like this (not sure if we need it to make scientific sense):
<details closed>
<summary>New section on `set_record_option` </summary>

<br>

Alternatively, you can also modify or append new options with
`set_record_option()`. Note that position dependent options must
be modified using the above method.

```{r}
est1
nmrec::set_record_option(est1, "CITER", value = 15) # Update CITER to 15
est1
nmrec::set_record_option(est1, "NOABORT", NULL) # Remove NOABORT
est1
nmrec::set_record_option(est1, "MAXEVAL", 10) # Add option MAXEVAL=10
est1
```
</details>

Reply:
For the "Getting Started" vignette, I'd say you can leave aside details about how `list1` is represented.  There are various other spots where entire strings are gobbled in (e.g., table's `by` or data's `format`).

The current vignette has a [section on modifying options](https://github.com/metrumresearchgroup/nmrec/blob/f79c7c3c5e053a93448d002c4334d7343c3944bf/vignettes/nmrec.Rmd#L113-L136).  You could extend that to point out that `set_record_option` exists to cover the common case of setting flag/value options and will adjust the option if present and add it otherwise.

Reply:
@kyleam 
> The current vignette has a [section on modifying options](https://github.com/metrumresearchgroup/nmrec/blob/f79c7c3c5e053a93448d002c4334d7343c3944bf/vignettes/nmrec.Rmd#L113-L136). You could extend that to point out that set_record_option exists to cover the common case of setting flag/value options and will adjust the option if present and add it otherwise.

Yes this is what I was thinking (and where I included it).

>For the "Getting Started" vignette, I'd say you can leave aside details about how list1 is represented. There are various other spots where entire strings are gobbled in (e.g., table's by or data's format).

We thought this should be documented a bit more somewhere, but doesn't have to be the vignette (could be a part of the `get_record_option` helper docs or something). I will remove it from the vignette for the time being though.